apps/employee-monitoring/src/components/modal/settings/system-logs/ViewSystemLogModal.tsx

```diff
@@ -20,61 +25,137 @@ const ViewSystemLogModal: FunctionComponent<ViewSystemLogModalProps> = ({
   closeModalAction,
   setModalState,
 }) => {
+  // useSWR
+  const {
+    data: swrSystemLogDetails,
+    isLoading: swrIsLoading,
+    error: swrError,
+  } = useSWR(modalState && rowData.id ? `/user-logs/${rowData.id}` : null, fetcherEMS, {
+    shouldRetryOnError: false,
```

Comment:
Remove this as the table view is deemed to be always updated real time while viewing.